.github/workflows/ets-from-source.yml

```
@@ -0,0 +1,83 @@
+name: Test with ETS packages from source
+
+on:
+  pull_request:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+  schedule:
+    # Every Friday at 00:00 UTC
+    - cron:  '0 0 * * 5'
+
+env:
+  INSTALL_EDM_VERSION: 3.2.3
+
+jobs:
+
+  # Test against EDM packages from source on Linux
+  test-with-edm:
+    strategy:
+      matrix:
+        os: ubuntu-latest
```

Looks like the workflow wasn't actually running because of this error. See https://github.com/enthought/chaco/actions/runs/979545563

```suggestion
        os: [ubuntu-latest]
```